### PR TITLE
Feat/spine bundle

### PR DESCRIPTION
**Pull request by fanmingfei**
### Please check if the PR fulfills these requirements

- [ ] The commit message follows our [guidelines](https://github.com/eva-engine/eva.js/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Other information:

### CODE REVIEW

File: packages/plugin-renderer-dragonbone/lib/system.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ const events = {
 
 const factory = dragonBones.PixiFactory.factory;
 
+resource.registerResourceType('DRAGONBONE')
```
Comment (terrykingcha) [inline packages/plugin-renderer-dragonbone/lib/system.ts:22]: 为啥不和registerInstance的api合在一起？

Comment (fanmingfei) [reply packages/plugin-renderer-dragonbone/lib/system.ts:22]: 因为，可能有些资源不需要registerInstance，但是addResource的时候资源类型肯定是要带的。

Comment (terrykingcha) [reply packages/plugin-renderer-dragonbone/lib/system.ts:22]: Done

**Comment by terrykingcha** [comment]
Done

**Review by terrykingcha — APPROVED** [review]

**Pull request merged**